### PR TITLE
Sidekiq Scheduler

### PR DESCRIPTION
**Pull request by cquiro**
## Summary

- Add sidekiq-scheduler gem to project
- Add send_rent_due_email method to RentsMailer
- Create RentDueNotifier worker
- Schedule RentDueNotifier to run every day at 9:00 am GMT-3

## Trello Card

https://trello.com/c/c0W6FnP9/41-notificar-al-usuario-cuando-un-rent-se-vence

## Screenshots

screen shot 2018-10-03 at 5 04 53 pm

screen shot 2018-10-03 at 5 05 04 pm

### CODE REVIEW

File: app/workers/rent_due_notifier.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class RentDueNotifier
+  include Sidekiq::Worker
+
+  def perform
+    Rent.where(id: rent_ids).find_each do |rent|
```
Comment (facucompe) [inline app/workers/rent_due_notifier.rb]: Why are you doing a where here? You already do it in `rent_ids` method

File: config/sidekiq.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+:schedule:
+  rent_due_notifier:
+    cron: '0 9 * * * Etc/GMT-3'
```
Comment (facucompe) [inline config/sidekiq.yml]: You can use `'0 9 * * * America/Santiago'`, it's more clear

**Review by facucompe — CHANGES_REQUESTED** [review]

File: app/workers/rent_due_notifier.rb (hunk 1 above)
Comment (cquiro) [reply app/workers/rent_due_notifier.rb]: @facucompe My bad! At some point during development I was returning rent_ids and using them in perform, then I realized it was not necessary and removed the `.ids` part in the rent_ids method, but forgot to change the rest.

File: app/mailers/rents_mailer.rb [hunk 3]
```diff
@@ -1,15 +1,26 @@
 class RentsMailer < ApplicationMailer
   def send_new_rent_email(rent_id)
-    @rent = Rent.find(rent_id)
-    @user = @rent.user
-    @book = @rent.book
+    get_rental_data(rent_id)
```
Comment (lmasello) [inline app/mailers/rents_mailer.rb]: Avoid using getters and setters in the names :) (https://github.com/rubocop-hq/ruby-style-guide#accessor_mutator_method_names)

**Review by lmasello — COMMENTED** [review]
Just a comment

**Review by facucompe — APPROVED** [review]

**Pull request merged**